dbusmock/templates/systemd.py

```diff
@@ -0,0 +1,106 @@
+'''systemd mock template
```

Comment:
Please adjust the commit message -- that still talks about -user and -system templates and the shared -base.

Reply:
Adjusted.